src/Frontend/src/views/throughputreport/setup/ConnectionSetupView.vue

```diff
@@ -0,0 +1,118 @@
+    </p>
+    <p v-else>No further configuration required.</p>
+  </div>
+  <template v-if="settingsInfo?.broker_settings.length ?? 0 > 0">
+    <div class="row configuration">
+      <div class="col-12">
+        <h4>Broker Settings</h4>
+        <p class="nogap">
+          Settings to ensure that usage data is being collected from {{ transportNameForInstructions() }}.<br />
+          Some settings can be automatically configured based on the current transport configuration, so if you have a <i style="color: green" class="fa fa-check"></i> above it means that ServiceControl has successfully connected to
+          {{ transportNameForInstructions() }}.
+        </p>
+        <ConfigurationCode :settings="settingsInfo?.broker_settings ?? []">
+          <template #configInstructions>
+            <div>Paste the settings above in the <code>ServiceControl.exe.config</code> file of the ServiceControl Error instance.</div>
+          </template>
+          <template #environmentVariableInstructions>
+            <div>Execute the above instructions in a terminal to set the environment variables, these variables need to be set for the user that in running the ServiceControl Error instance.</div>
+          </template>
+        </ConfigurationCode>
+      </div>
+    </div>
+  </template>
+  <template v-if="useIsMonitoringEnabled()">
+    <div class="row configuration">
+      <div class="col-12">
+        <h4>ServiceControl Settings</h4>
+        <p class="nogap">
+          Settings to ensure that usage data is being collected from the Monitoring instance.<br />
+          For more information read the <a href="https://docs.particular.net/servicecontrol/creating-config-file">ServiceControl</a> settings documentation.
+        </p>
+        <ConfigurationCode :settings="settingsInfo?.service_control_settings ?? []">
+          <template #configInstructions>
+            <div>Paste the settings above in the <code>ServiceControl.exe.config</code> file of the ServiceControl Error instance.</div>
+          </template>
+          <template #environmentVariableInstructions>
+            <div>Execute the above instructions in a terminal to set the environment variables, these variables need to be set for the user that in running the ServiceControl Error instance.</div>
+          </template>
+        </ConfigurationCode>
+      </div>
+    </div>
+    <div class="row configuration">
+      <div class="col-12">
+        <h4>Monitoring Settings</h4>
+        <p class="nogap">
+          Settings to ensure that usage data is being collected from the Monitoring instance.<br />
+          These settings do not need to be modified unless MSMQ transport is used with the Monitoring instance installed on a different machine to the ServiceControl Error instance.<br />
+          For more information read the <a href="https://docs.particular.net/servicecontrol/monitoring-instances/installation/creating-config-file">Monitoring</a> settings documentation.
```

Comment:
This is a new link - need to verify it once it's running

```suggestion
          For more information read the <a href="https://docs.particular.net/servicecontrol/monitoring-instances/installation/creating-config-file#usage-reporting">Monitoring</a> settings documentation.
```